suite-native/formatters/src/components/EthereumTokenAmountFormatter.tsx

```diff
@@ -11,22 +11,37 @@ import { AmountText } from './AmountText';
 type EthereumTokenAmountFormatterProps = {
     ethereumToken: EthereumTokenSymbol;
     isDiscreetText?: boolean;
+    decimals?: number;
 } & FormatterProps<number | string> &
     TextProps;
 
 export const EthereumTokenAmountFormatter = ({
     value,
     ethereumToken,
     isDiscreetText = true,
+    decimals = 0,
+    variant = 'hint',
+    color = 'textSubdued',
     ...rest
 }: EthereumTokenAmountFormatterProps) => {
-    const formattedValue = localizeNumber(value);
+    const formattedValue = localizeNumber(Number(value) / 10 ** decimals);
```

Comment:
please write a comment why 10 for future so that it's not guessing. Even better would be to use const with appropriate naming

Reply:
convert function created 4d5c05278

Reply:
Still using 10 instead of const. If you don't use constant, write a comment why 10

Reply:
It is ten because we are using a decimal system. This (`Number(value) / 10 ** decimals)`) operation is a simple shift of `n` decimal points. More here: https://www.mathsisfun.com/numbers/scientific-notation.html. IMHO it's a very basic operation, putting a const there would make it only more confusing. I believe that the function name is self-explanatory.

Reply:
the code you are refering to is outdated. Since https://github.com/trezor/trezor-suite/commit/4d5c05278740cdd3d9e85981c1257cf8feac0a25  it looks like: 

```tsx
...
const decimalValue = convertTokenValueToDecimal(value, decimals);
const formattedValue = localizeNumber(decimalValue);
...
```